bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

```diff
@@ -385,4 +385,15 @@ default String getCustomFunctionUrl() {
         return null;
     }
 
+    /**
+     * Returns the auto save configuration
+     *
+     * @return auto save configuration
+     * @since com.adobe.cq.forms.core.components.models.form 5.10.0
+     */
+    @JsonIgnore
+    default AutoSaveConfiguration getAutoSaveConfig() {
```

Comment:
Since there is a change in interface, update minor version and update package-info. Also update java doc